Thu Nov 16 16:44:34 CET 2006  hanwen@lilypond.org
  * check for cwait fsync pipe popen spawnv system
diff -rN -u old-Python-2.5.cross/configure.in new-Python-2.5.cross/configure.in
--- old-Python-2.5.cross/configure.in	2006-11-16 17:47:46.000000000 +0100
+++ new-Python-2.5.cross/configure.in	2006-11-16 17:47:46.000000000 +0100
@@ -2536,6 +2536,8 @@
   )
 )
 
+AC_CHECK_FUNCS(cwait fsync pipe popen spawnv system)
+
 AC_MSG_CHECKING(for major, minor, and makedev)
 AC_TRY_LINK([
 #if defined(MAJOR_IN_MKDEV)

